care/facility/api/serializers/patient.py

```diff
@@ -457,6 +460,12 @@ class Meta:
 class ReplyToPatientNoteSerializer(serializers.ModelSerializer):
     id = serializers.CharField(source="external_id", read_only=True)
     created_by_object = UserBaseMinimumSerializer(source="created_by", read_only=True)
+    reply_to_object = serializers.SerializerMethodField()
+
+    def get_reply_to_object(self, obj):
+        if obj.reply_to:
+            return ReplyToPatientNoteSerializer(obj.reply_to).data
+        return None
```

Comment:
```suggestion
    reply_to_object = ReplyToPatientNoteSerializer(source="reply_to", read_only=True)
```

Reply:
i am refering to same serializer

Reply:
@UdaySagar-Git Yes, but what @rithviknishad suggested is cleaner, and avoids unwanted extra code.

Reply:
Adding a parent_note field to the note made it no longer required, and having just the reply_to ID was sufficient for it to work, since the note will always be present in child_notes.

made frontend changes accordingly